install.rst

```diff
@@ -225,13 +252,16 @@ Git repositories hosted at https://github.com/zeek:
 
 .. code-block:: console
 
-    git clone --recursive https://github.com/zeek/zeek
+    git clone --recurse-submodules https://github.com/zeek/zeek
 
 .. note:: If you choose to clone the ``zeek`` repository
    non-recursively for a "minimal Zeek experience", be aware that
    compiling it depends on several of the other submodules as well, so
    you'll likely have to build/install those independently first.
 
+Configuring and Building
+------------------------
+
 The typical way to build and install from source is (for more options,
```

Comment:
You can remove the `for more options` sidebar here. It's mentioned later already.

Reply:
Done